django-nonrel/ocl/integration_tests/tests/all.py

```diff
@@ -2108,6 +2108,95 @@ def test_post(self):
         # self.assertEquals(response['Last-Updated-Timezone'], 'America/New_York')
 
     @mock_s3
+    def test_get_version_ocl_processing(self):
+        source = Source(
+            name='source',
+            mnemonic='source',
+            full_name='Source One',
+            source_type='Dictionary',
+            public_access=ACCESS_TYPE_EDIT,
+            default_locale='en',
+            supported_locales=['en'],
+            website='www.source1.com',
+            description='This is the first test source'
+        )
+
+        kwargs = {
+            'parent_resource': self.org1
+        }
+        Source.persist_new(source, self.user1, **kwargs)
+
+        source_version = SourceVersion(
+            name='version1',
+            mnemonic='version1',
+            versioned_object=source,
+            released=True,
+            created_by=self.user1,
+            updated_by=self.user1,
+            _ocl_processing=True
+        )
+        source_version.full_clean()
+        source_version.save()
+        kwargs = {'parent_resource': source}
+        (concept1, _) = create_concept(mnemonic='concept1', user=self.user1, source=source)
+        c = Client()
+        c.post('/login/', {'username': 'user1', 'password': 'user1'})
+
+        kwargs = {
+            'org': self.org1.mnemonic,
+            'source': source.mnemonic,
+            'version': 'version1'
+        }
+        response = c.get(reverse('sourceversion-export', kwargs=kwargs))
+        self.assertEquals(response.status_code, 409)
```

Comment:
Is this going to be a stable test? Is it possible that the export is ready by the time you do get and it returns 204?

Reply:
There's no export, it's creating the version with _ocl_processing = True, as though the version is still being created.  Therefore, it should always respond with a 409.

Reply:
Okay!